decidim-conferences/app/views/devise/mailer/join_conference.text.erb

```diff
@@ -2,7 +2,7 @@
 
 <%= t("decidim.conferences.admin.invite_join_conference_mailer.invite.invited_you_to_join_a_conference", invited_by: @resource.invited_by.name, application: @resource.organization.name) %>
```

Comment:
Should we change this as we already changed the HTML version? 

```suggestion
<%= t("decidim.conferences.admin.invite_join_conference_mailer.invite.invited_user_to_join_a_conference", invited_by: @resource.invited_by.name, application: @resource.organization.name) %>
```

On a related note, does it make sense to have this file at all? Shouldn't mails plain text version be handled by [premailer](https://github.com/decidim/decidim/blob/develop/decidim-core/lib/premailer/adapter/decidim.rb)? (I see that indeed we have a couple of plain text version mailers, I think we can handle that on other PR)

Reply:
This translation is different than `invited_user_to_join_a_conference`.

[I changed it to `invited_existing_user_to_join_a_conference`](https://github.com/decidim/decidim/pull/9664/commits/e7bbb3361f606fd3f553c93629c36a3698c48494).

Reply:
Great!